src/javascript/app/base/footer.js

```diff
@@ -0,0 +1,20 @@
+const Footer = (() => {
+    const displayNotification = (message) => {
+        const status_notification = $('#status_notification');
+        const status_message_text = $('#status_notification_text');
+        const close_icon = $('#status_notification_close');
```

Comment:
for jquery variables, please start the name with `$`, e.g. `$status_notification` etc